plugins/modules/net_tools/nmcli.py

```diff
@@ -144,9 +144,10 @@
     ip6:
         description:
             - The IPv6 address to this interface.
```

Comment:
Nit, but I would update the wording to match ip4 _List of IPv6 addresses to this interface._

Reply:
Done